act-rules-format.bs

```diff
@@ -170,41 +192,89 @@ When all expectations are true for a test target, the test target `passed` the r
+  <blockquote>
+    <p>For each test target, the outcome of one of the following rules is `passed`:</p>
+    <ul>
+      <li>Video elements have an audio description</li>
+      <li>Video elements have a media alternative</li>
+      <li>Video elements is media alternative</li>
+    </ul>
+  </blockquote>
+</div>
+
 
-Note that rules in a group MAY have different [applicability](#test-applicability). Because of this, not every element applicable within the group is tested by every rule. Rules MAY also be disabled during a test run due to accessibility support concerns. See [Accessibility Support](#acc-support) for details.
+Limitations, Assumptions or Exceptions {#structure-limitations-assumptions-exceptions}
+======================================================================================
+
+An ACT Rule MUST list any limitations, assumptions or any exceptions for the test, the test environment, technologies being used or the subject being tested. For example, a rule that would partially test [WCAG 2.0 Success Criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast) based on the inspection of CSS properties could state that it is only applicable to HTML text content stylable with CSS, and that the rule does not support images of text.
+
+Sometimes there are multiple plausible ways that an accessibility requirement can be interpreted. For instance, it is not immediately obvious if emoji characters should be considered "text" or "non-text content" under WCAG 2.0. Whatever the interpretation is, this MUST be documented in the rule.
+
+
+Accessibility Support {#acc-support}
+====================================
+
+ACT Rules are designed to test the conformance of content using web technologies to accessibility requirements. However, not every feature of a web technology is implemented in all assistive technologies or user agents that a website may need to support. The concept of [accessibility supported](https://www.w3.org/TR/WCAG20/#accessibility-supporteddef) use of a Web technology is defined in WCAG [[WCAG]]. Because of this, ACT Rules are not necessarily applicable in all test scenarios. For instance, a web page that has to work in assistive technologies that have no WAI-ARIA [[WAI-ARIA]] support, wouldn’t be tested with an ACT Rule that relies on WAI-ARIA support, since this could lead to false positive results.
+
+Even within a [composed rule](#composed-rule-outline), some [atomic rules](#atomic-rule-outline) may not always be applicable. This leaves one fewer solution for passing that particular atomic rule. To support users of ACT Rules in properly defining the [accessibility support baseline](https://www.w3.org/TR/WCAG-EM/#step1c) in their test scenarios, an ACT Rule SHOULD include a warning if there are significant accessibility support concerns known about a rule.
 
```

Comment:
Second sentence - I think you mean there are fewer solutions for passing the composed rule, right?